### PR TITLE
Fix warning export loadingIndicator not found when run build

### LINKED ISSUE #1385: [bug]: loadingIndicator import in cartOptions.js is invalid
**Describe the bug**
We updated the loading indicator to export a `fullPageLoadingIndicator`. We missed one spot in merging in `cartOptions`.

```
WARNING in ./src/components/MiniCart/cartOptions.js 75:14-30
"export 'loadingIndicator' was not found in 'src/components/LoadingIndicator'
 @ ./src/components/MiniCart/editItem.js
 @ ./src/components/MiniCart/body.js
 @ ./src/components/MiniCart/miniCart.js
 @ ./src/components/MiniCart/container.js
 @ ./src/components/MiniCart/index.js
 @ ./src/components/App/app.js
 @ ./src/components/App/container.js
 @ ./src/components/App/index.js
 @ ./src/index.js
```

**To Reproduce**
Check out develop and `yarn install` and `yarn build`. You'll see the warning.
**Expected behavior**

**Possible solutions**
Update the imports to the proper name and double check whether a full page indicator is actually valid in cartOptions now.

**Please complete the following device information:**
 - Device [e.g. iPhone6, PC, Mac, Pixel3]:
 - OS [e.g. iOS8.1, Windows 10]:
 - Browser [e.g. Chrome, Safari]:
 - Browser Version [e.g. 22]:
 - Magento Version:
 - PWA Studio Version:
 - NPM version `npm -v`:
 - Node Version `node -v`:

**Please let us know what packages this bug is in regards to:**
- [x] `venia-concept`
- [ ] `pwa-buildpack`
- [ ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

### PR DESCRIPTION
**Pull request by mrtuvn**
# Description
Uses the proper loading indicator and creates a static component for use as the suspense fallback.

# Verification Steps
1. Add item to cart
2. In cart, click edit item.
3. Cart options load and display.

Will close #1385 along side #1388

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=1391) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=1391) before we can accept your contribution.<br/>

File: packages/venia-concept/src/components/MiniCart/cartOptions.js [hunk 1]
```diff
@@ -4,9 +4,7 @@ import { Form } from 'informed';
 import { Price } from '@magento/peregrine';
 
 import { mergeClasses } from 'src/classify';
-import LoadingIndicator, {
-    loadingIndicator
```
Comment (sirugh) [inline packages/venia-concept/src/components/MiniCart/cartOptions.js]: There is still a reference to `loadingIndicator` used on line 84 as a suspense fallback. Here's the thing -- we changed the `LoadingIndicator` component to export two things: a static indicator that renders as a full page (`fullPageLoadingIndicator`) or a `LoadingIndicator` component that accepts styles. I'm not sure the cart will need the full page loading indicator so you will probably want to create an inline static one here using `LoadingIndicator and pass it to the component on line 84.

For example:

```jsx
const loadingIndicator = (<LoadingIndicator><span>{'Fetching Options...'}</span></LoadingIndicator>);

...

<Suspense fallback={loadingIndicator}...
```

Here's an example of how we do it in `editItem`: https://github.com/magento-research/pwa-studio/blob/develop/packages/venia-concept/src/components/MiniCart/editItem.js#L11-L13

**Comment by sirugh** [comment]
I left you a comment -- you will also want to make sure to fill out the PR template in full and sign the CLA. Thanks!

File: packages/venia-concept/src/components/MiniCart/cartOptions.js (hunk 1 above)
Comment (mrtuvn) [reply packages/venia-concept/src/components/MiniCart/cartOptions.js]: yeah i see. Updated PR

**Comment by sirugh** [comment]
Great! Thanks for making the change. Please sign the CLA so we can get this merged ASAP as it's a high priority issue for us right now.

**Pull request merged**